### PR TITLE
Fix go.mod and go.sum in tools to fix dependabot alerts

### PR DESCRIPTION
**Pull request by prateekgogia**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: operator/tools/go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
 	github.com/fzipp/gocyclo v0.0.0-20150627053110-6acd4345c835
 	github.com/golangci/golangci-lint v1.31.0
-	github.com/google/ko v0.8.3
+	github.com/google/ko v0.9.3
```
Comment (ellistarn) [inline operator/tools/go.mod:9]: v0.10.0?

**Review by ellistarn — DISMISSED** [review]

**Comment by ellistarn** [comment]
Consider something like: https://github.com/aws/karpenter/blob/main/.github/dependabot.yml

File: operator/tools/go.mod (hunk 1 above)
Comment (prateekgogia) [reply operator/tools/go.mod]: Had to upgrade all the tools, just ko wasn't enough

**Comment by prateekgogia** [comment]
Once PR https://github.com/awslabs/kubernetes-iteration-toolkit/pull/135 is merged `go version` will be updated in github actions to `1.17.1` that should fix this build.

**Review by ellistarn — APPROVED** [review]

**Review by ellistarn — APPROVED** [review]

**Pull request merged**